go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -279,3 +281,41 @@ func runOvnKube(ctx *cli.Context) error {
 
 	return nil
 }
+
+// parseClusterSubnetEntries returns the parsed set of CIDRNetworkEntries passed by the user on the command line
+// These entries define the clusters network space by specifying a set of CIDR and netmaskas the SDN can allocate
+// addresses from.
+func parseClusterSubnetEntries(clusterSubnetCmd string) ([]ovncluster.CIDRNetworkEntry, error) {
```

Comment:
Oh, I just remembered that, in the code for the CNI host-local allocator, we detect if the administrator accidentally configures overlapping subnets.

There's a simple implementation here: https://github.com/containernetworking/plugins/blob/master/plugins/ipam/host-local/backend/allocator/range.go#L131

Do you want to add this? It could be a separate PR, if you like.

Reply:
@squeed I'm fine with it being a follow-up PR.

Reply:
@squeed Jacob added code and testcases to do this.